### ISSUE TITLE
[BUG] titleStyle and descriptionStyle not configurable for individual toasts

### ISSUE THREAD

**Issue by tomkelsey** [gunnartorfis/sonner-native#178]
**Describe the bug**
It looks like you can set the default title and description styles using ```<Toaster>``` but that you can't configure this on per-toast level? I'm not sure if this is intended or a bug?

It would be great if you could customise on a per toast level too e.g.

```
toast('My Toast Message', {
titleStyle: {
color: 'red'
})
```

**Expected behavior**
The title text is red

PS. Thanks for your efforts with this library, it's great!

**Comment by gunnartorfis** [comment]
I agree, changing the style on a per toast level would be a great option! I'll take a look at this soon.
